packages/sensors_plus/sensors_plus/example/integration_test/sensors_plus_test.dart

```diff
@@ -22,4 +22,64 @@ void main() {
     });
     expect(await completer.future, isNotNull);
   });
+
+  testWidgets('Can subscript to gyroscopeEvents and get non-null events',
+      (WidgetTester tester) async {
+    final completer = Completer<GyroscopeEvent>();
+    StreamSubscription<GyroscopeEvent> subscription;
+    subscription = gyroscopeEvents.listen((GyroscopeEvent event) {
+      completer.complete(event);
+      subscription.cancel();
+    });
+    expect(await completer.future, isNotNull);
+  });
+
+  testWidgets('Can subscript to gyroscopeEvents and get expected events',
```

Comment:
I am not sure if I get the idea of this test, you are comparing at the end the types of the completer, which is something that is not going to change.